v2.0/upgrade-cockroach-version.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Upgrade to CockroachDB v2.0
+title: Upgrade to CockroachDB v2.0.3
 summary: Learn how to upgrade your CockroachDB cluster to a new version.
 toc: false
 toc_not_nested: true
 ---
 
 Because of CockroachDB's [multi-active availability](multi-active-availability.html) design, you can perform a "rolling upgrade" of your CockroachDB cluster. This means that you can upgrade nodes one at a time without interrupting the cluster's overall health and operations.
 
-{{site.data.alerts.callout_info}}This page shows you how to upgrade to v2.0 from v1.1.x, or from any patch release in the v2.0.x series. To upgrade within the v1.1.x series, see <a href="https://www.cockroachlabs.com/docs/v1.1/upgrade-cockroach-version.html">the v1.1 version of this page</a>.{{site.data.alerts.end}}
+{{site.data.alerts.callout_info}}This page shows you how to upgrade to v2.0.3 from v1.1.x, or from any patch release in the v2.0.x series. To upgrade within the v1.1.x series, see <a href="https://www.cockroachlabs.com/docs/v1.1/upgrade-cockroach-version.html">the v1.1 version of this page</a>.{{site.data.alerts.end}}
 
 <div id="toc"></div>
 
 ## Step 1. Verify that you can upgrade
 
-When upgrading, you can skip patch releases, **but you cannot skip full releases**. Therefore, if you are upgrading from v1.0.x to v2.0:
+When upgrading, you can skip patch releases, **but you cannot skip full releases**. Therefore, if you are upgrading from v1.0.x to v2.0.3:
```

Comment:
Wherever you've manually changed this, would you mind trying to use the following Jekyll variable instead?

`{{ page.release_info.version }}`

That should get the `release_info` > `v2.0` > `version` from `_config.yml`. I'm not sure this will work in all places on the page (specifically unsure about using this inside a callout and in the title), but it's worth a try.

If it doesn't work, this LGTM. Thank you for taking the initiative. You're very encouraged to do so whenever you see the need!